acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -8,60 +8,164 @@
 
 import UIKit
 import IGListKit
+import SnapKit
 
 protocol DidSelectOptionDelegate: class {
     func deselectOption(at index: Int)
     func selectOption(at index: Int, type: OrderCustomizationOptionType)
 }
 
-class BowlOrderDetailViewController: UIViewController {
+class BowlOrderDetailViewController: UIViewController, UIGestureRecognizerDelegate {
     
     // MARK: View vars
+    var addToCartActionTabView: ActionTabView!
+    var backButtonImageView: UIImageView!
+    var bottomFillerRect: UIView!
+    var collectionBottomFillerRect: UIView!
     var collectionView: UICollectionView!
     var listAdapter: ListAdapter!
+    var titleLabel: UILabel!
+    
+    // MARK: Gesture recognizers
+    var addToCartTapGestureRecognizer: UITapGestureRecognizer!
+    var backButtonTapGestureRecognizer: UITapGestureRecognizer!
     
     // MARK: Data
     var baseOptions: [OrderCustomizationOption]!
     var bowlItem: BowlItem!
     var toppingOptions: [OrderCustomizationOption]!
     
+    // MARK: Constraint Constants
+    let addToCartActionTabViewHeight = 55
+    let backButtonLeadingOffset = 18
+    let backButtonHeight = 15
+    let backButtonWidth = 8.7
+    let emptyItemHeight: CGFloat = 156
+    let titleLabelTopOffset = 5.5
+    
     // MARK: View Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .white
+        view.backgroundColor = .acaiColdGray
+//        navigationController?.navigationBar.shadowImage = UIImage()
+//        navigationController?.navigationBar.barTintColor = .clear
+//        navigationController?.navigationBar.tintColor = .white
+//        navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white, NSAttributedString.Key.font: UIFont.avenirNextBold.withSize(24)]
+//        navigationController?.navigationBar.backgroundColor = .clear
```

Comment:
prefer deleting code rather than commenting it out

Reply:
+1